README.md

```diff
@@ -4,12 +4,28 @@ A open source screen reader for android.
 ## Build
 In development, choose the *debug build type*, because for the *release build type*, it's necessary to create a [keystore](https://developer.android.com/training/articles/keystore) and specify its properties in the `keystore.properties` file (use `keystore.properties.example` as a base).
 
-### IDE
-To build you need [Android Studio](https://developer.android.com/studio/releases) (Flamingo or later).
+### Android Studio
 
-1. Clone the repository to your local machine.
-2. Open the project in the IDE.
-3. Run the **app** module.
+1. Download and install [Android Studio](https://developer.android.com/studio).
+2. Clone the repository to your local machine.
+3. Open the project in the Android Studio.
+4. Run the **app** module.
+
+### IntelliJ IDEA
+
+1. Download and install [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+2. Clone the repository to your local machine.
+3. Open the project in the IntelliJ IDEA.
+4. Run the **app** module.
+
+### Command line
+1. Download and install [JDK 17](https://www.oracle.com/java/technologies/downloads/#java17) or newer.
```

Comment:
I would prefer Adopt OpenJDK (`1. Download and install [Adoptium OpenJDK 17](https://adoptium.net/download/) or newer.`).

OpenJDK is free software and has binaries for all major systems and distributions and is available from all know package managers.

Reply:
ok